components/products.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    fetch('https://panda-market-api.vercel.app/api/products')
```

Comment:
API링크가 틀려서 의도하신 데이터가 안올거같아요 이렇게 수정해보시면 될거에요
```suggestion
    fetch('https://panda-market-api.vercel.app/products')
```